src/jobstats.rs

```diff
@@ -170,110 +170,151 @@ type JobStatMdtPromInst<'a> = (
     PrometheusInstance<'a, i64, Yes>,
     PrometheusInstance<'a, i64, Yes>,
     PrometheusInstance<'a, i64, Yes>,
+    Option<PrometheusInstance<'a, i64, Yes>>,
+    Option<PrometheusInstance<'a, i64, Yes>>,
 );
 
 fn jobstatmdt_inst<'a>(
     x: &'a JobStatMdt,
     kind: &'a str,
     target: &'a str,
 ) -> JobStatMdtPromInst<'a> {
+    let JobStatMdt {
+        open,
+        close,
+        mknod,
+        link,
+        unlink,
+        mkdir,
+        rmdir,
+        rename,
+        getattr,
+        setattr,
+        getxattr,
+        setxattr,
+        statfs,
+        sync,
+        samedir_rename,
+        crossdir_rename,
+        job_id,
+        parallel_rename_dir,
+        parallel_rename_file,
+        ..
```

Comment:
```suggestion
```

List all fields and use `unused_field: _` to ignore them.
This way we have an exhaustive list and will fail to compile if we are not explicitly handling a new field (ignoring it or using it)

Reply:
Done :)